packages/nimble-components/src/table/components/row/tests/table-row.spec.ts

```diff
@@ -49,8 +49,7 @@ describe('TableRow', () => {
 
```

Comment:
nit: I'd argue the test name doesn't really match what it's asserting anymore. Maybe something more like:
`    it('column cellViewTag is applied to generated cells', async () => {`

Reply:
Updated